package.json

```diff
@@ -295,13 +298,16 @@
     "eslint-plugin-react": "7.20.0",
     "eslint-plugin-react-hooks": "4.0.2",
     "express": "4.17.1",
+    "filesize": "^6.4.0",
     "glob": "7.1.6",
     "hamburgers": "1.1.3",
     "handlebars": "4.7.7",
     "history": "4.10.1",
+    "hoist-non-react-statics": "^3.3.2",
     "http-proxy-middleware": "1.0.4",
     "husky": "4.2.5",
     "identity-obj-proxy": "3.0.0",
+    "immutable": "^4.0.0-rc.12",
```

Comment:
This is a hard dependency for draftjs I pin it via "resolutions" but it has its own problems, like the resolutions are not passed to projects. However, I don't know if draftjs works with bleeding edge versions of immutable.

Reply:
I've pinned it to 3.8.2. But I get this message when running yarn:

```
warning Resolution field "immutable@3.8.2" is incompatible with requested version "immutable@~3.7.4"
```